common/services/prismic/utils.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import moment from 'moment';
+import {Predicates} from 'prismic-javascript';
+import {london} from '../../utils/format-date';
+import {getNextWeekendDateRange} from '../../utils/dates';
+
+export type DateRangeString =
+  | 'today'
+  | 'this-weekend'
+  | 'current-and-coming-up'
+  | 'past'
+  | 'current'
+  | 'coming-up'
+  | 'this-weekend'
+  | 'this-week';
+export function getDateRangePredicatesFromString(
+  rangeString: ?DateRangeString,
+  startField: string,
+  endField: string
+): Predicates[] {
+  const now = london(new Date());
+  const startOfDay = moment().startOf('day');
+  const endOfDay = moment().endOf('day');
+  const weekendDateRange = getNextWeekendDateRange(now);
+
+  const range =
+    rangeString === 'current' ? [
+      Predicates.dateBefore(startField, endOfDay.toDate()),
+      Predicates.dateAfter(endField, endOfDay.toDate())
```

Comment:
Think this should be Predicates.dateAfter(endField, startOfDay.toDate()) otherwise it would miss something that finished on the current day. But this is what 'today' is doing so not sure what the difference between the to is/should be?